system/security/LdapSecurity/aci.cpp

```diff
@@ -972,7 +976,7 @@ void AciProcessor::cacheSid(const char* name, int len, const void* sidbuf)
 void AciProcessor::lookupSid(const char* act_name, MemoryBuffer& act_sid, ACT_TYPE acttype)
```

Comment:
If we don't need this method, why don't we remove it? Is it declared in an interface/base class? If so do we want to clean up the interface/base class?